README.md

```diff
@@ -25,6 +25,7 @@ These are all the individual documents describing each small piece of protocol t
 | [18][18] | Payer identity in `payRequest` protocol.                    | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [cliché][cliche], [OBW][obw], [ZBD Discord][zbd], [ZBD Telegram][zbd] |
 | [19][19] | Pay link discoverable from withdraw link.                   | [Blixt][blixt], [CoinCorner][coincorner], [OBW][obw], [LNbits][lnbits] |
 | [20][20] | Long payment description for pay protocol.                  | [Alby][alby], [BitBanana][bitbanana], [Blixt][blixt], [Clams][clams], [cliché][cliche], [Mash][mash], [OneKey][onekey], [Phoenix][phoenix] |
+| [21][21] | `verify` invoice payments                                   | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news] |
```

Comment:
```suggestion
| [21][21] | `verify` invoice payments                                   | [Alby][alby], [Mutiny][mutiny], [Stacker.News][stacker.news], [Zaprite][zaprite] |
```

Zaprite also supports this already too. Would need to add them to the links below too. 

```
[zaprite]: https://zaprite.com
```